editor/export/editor_export_platform_apple_embedded.cpp

```diff
@@ -192,9 +192,10 @@ String EditorExportPlatformAppleEmbedded::get_export_option_warning(const Editor
 			}
 		} else if (p_name == "shader_baker/enabled") {
```

Comment:
The check could be here directly instead of repeating it on each inner branch:
```suggestion
		} else if (p_name == "shader_baker/enabled" && p_preset->get("shader_baker/enabled")) {
```

Reply:
I followed the convention of matching only the p_name first and then performing additional checks in the condition block.

Reply:
done